js/main.js

```diff
@@ -96,3 +128,27 @@ const $newButton = document.querySelector('.new-button');
 $newButton.addEventListener('click', function (event) {
   viewSwap('entry-form');
 });
+
+const $entryTitle = document.querySelector('#title');
+const $entryNotes = document.querySelector('#notes');
+const $h1 = document.querySelector('h1');
+
+$ul.addEventListener('click', function pencilClick(event) {
+  if (event.target.tagName === 'I') {
+    const dataEntryId = event.target
+      .closest('li')
+      .getAttribute('data-entry-id');
+
+    viewSwap('entry-form');
```

Comment:
I would call this after the loop, that way we swap after the inputs were filled in